### PR TITLE
CAN ISO-TP progress (+unit-tests fix)

### PR DESCRIPTION
**Pull request by andreika-git**

### CODE REVIEW

File: firmware/hw_layer/drivers/can/can_msg_tx.cpp [hunk 1]
```diff
@@ -87,20 +90,6 @@ void CanTxMessage::setBit(size_t byteIdx, size_t bitIdx) {
 	m_frame.data8[byteIdx] |= 1 << bitIdx;
 }
 
-#else
```
Comment (mck1117) [inline firmware/hw_layer/drivers/can/can_msg_tx.cpp:90]: why this change?

Comment (rusefillc) [reply firmware/hw_layer/drivers/can/can_msg_tx.cpp:90]: if we can remove code and tests are still passing it must be unneeded code?

Comment (mck1117) [reply firmware/hw_layer/drivers/can/can_msg_tx.cpp:90]: but the code didn't get removed, it just got #ifdef'd in every single function instead of all at once

Comment (rusefillc) [reply firmware/hw_layer/drivers/can/can_msg_tx.cpp:90]: @mck1117 please forgive me I was excited to try if this would have fixed PCAN. CAN defines are a mess at the moment, this is technical debt right now while I am excited about getting PCAN ready.
